### PR TITLE
Update user.controller.ts

### PR DESCRIPTION
**Pull request by fleventy-5**
Hey I have commented all the module import in user.controller.ts except the service import. I'm new to open source so please tell me if i made some mistakes. Thanks 
Issue: #71

### CODE REVIEW

File: src/user/user.controller.ts [hunk 1]
```diff
@@ -11,10 +11,10 @@ import {
...
 } from '@nestjs/common';
 import { UserService } from './user.service';
-import { CreateUserDto } from './dto/create-user.dto';
-import { ApiBearerAuth, ApiTags } from '@nestjs/swagger';
-import { JwtAuthGuard } from '../auth/guards/jwt.guard';
-import { AuthorizedUser } from './interface/user.interface';
+//import { CreateUserDto } from './dto/create-user.dto';
+//import { ApiBearerAuth, ApiTags } from '@nestjs/swagger';
+//import { JwtAuthGuard } from '../auth/guards/jwt.guard';
+//import { AuthorizedUser } from './interface/user.interface';
```
Comment (YashKumarVerma) [inline src/user/user.controller.ts:17]: these lines are not to be removed.

**Review by YashKumarVerma — CHANGES_REQUESTED** [review]

File: src/user/user.controller.ts (hunk 1 above)
Comment (fleventy-5) [reply src/user/user.controller.ts:17]: Ok sorry I thought I have to remove those modules too
So I have to remove only those methods for delete id or editing I'd etc?

Comment (YashKumarVerma) [reply src/user/user.controller.ts:17]: there are few functions (see issue body) that should be moved to dashboard service.

**Comment by fleventy-5** [comment]
Hey @YashKumarVerma can you assign this to someone else I'm having trouble understanding the structure of typescript projects.
Sorry for wasting your time. Thanks

**Comment by fleventy-5** [comment]
I'll try looking for other issues that are easier.

**Pull request closed without merging**